_collections/_news/2024-12-14-2024-25s-semester-1-overview.md

```diff
@@ -0,0 +1,151 @@
+have the **second-highest membership count in the entire Guild**, standing at 404 (not found) members! That number has
+only been seen once previously in the second semester, but CSS has pre-emptively smashed it out of the park.
+
+![A graph of our membership count so far compared to previous years.](/assets/images/2024-semester-1/membership-numbers.png)
+
+That's not even the full picture though, since that number has only accounted for standard memberships - if anyone has
+been at UoB since 2022, there were a series of changes to how external memberships worked (these are non-students and
+staff who want to be part of a society), lengthening the process and adding extra roadblocks which has since deterred
+alumni students and others from joining a society as an external member. However, this year, we've had 4 external members
+join us, leading to a total of 408 members - which is the *exact* membership count we hit at the end of 2022/23 and the 
+highest number of members CSS has seen in its 20+ years of history.
+
+This was not a record we aimed to reach this year - as mentioned in the EGM, our original goal was to help bring awareness
+to CSS and what we do early on, then allow people to settle into the society in their own time and if they choose to do
+so; and that is still true today. However, it's still left us on committee lost for words, and I think it speaks for the
+effort that they've all put in this semester. Thus, I want to give my most sincere thank you to the entirety of CSS
+committee this semester for making the impossible possible, putting everything they have into what they love, and being
+an unshakable team that can withstand even the toughest of tasks.
+
+But of course, let's not forget the reason we're actually in this position - it's thanks to you. CSS has maintained its
+unique and eccentric community throughout the years, but it wouldn't be that way without the members that make up that
+community. Thus, whether you've given a workshop, joined us in events or simply hanged around in Discord, CSS would not 
+be the same without you - thank you for making this our best semester in CSS' history. 💙
+
+# What's next
+As I round off this newsletter, I'd like to give you a brief glimpse into what we're looking to prepare next semester. 
+
+We've already started planning and even finalisation of our larger events, such as our trip to the National Museum of
+Computing, the glamorous ball, and the AGM to elect the entirety of committee for the new year. However, we also have a
+mystery event that we are currently working with the school on, as well as a few other societies to participate in. It's
+been a long while since we've seen this kind of event at UoB, but perhaps some events we've run this semester and will
+run the next semester too may give you a small idea... oh yeah, and calling for peace in the chaos of war against Bad
+Film.
```

Comment:
```suggestion
We've already started planning our larger events, such as our trip to the National Museum of
Computing, the glamorous Ball, and the AGM to elect the entirety of committee for the new year. However, we also have a
mystery event that we are currently working with the school on, as well as a few other societies to participate in. It's
been a long while since we've seen this kind of event at UoB, but perhaps some events we've run this semester and will
run the next semester too may give you a small idea... oh yeah, and calling for peace in the chaos of war against Bad
Film.
```